components/TxForm.tsx

```diff
@@ -1,34 +1,143 @@
 import useNotifications from "@/hooks/useNotifications";
+import useShieldPrivateKey from "@/hooks/useShieldPrivateKey";
 import { Button } from "@chakra-ui/button";
 import { FormControl, FormLabel } from "@chakra-ui/form-control";
 import { Input, InputGroup, InputRightElement } from "@chakra-ui/input";
 import { Box, Flex } from "@chakra-ui/layout";
 import { Select } from "@chakra-ui/select";
+import {
+  gasEstimateForShield,
+  getRailgunSmartWalletContractForNetwork,
+  populateShield,
+} from "@railgun-community/quickstart";
+import {
+  RailgunERC20AmountRecipient,
+  NetworkName,
+  TransactionGasDetailsSerialized,
+  EVMGasType,
+  NETWORK_CONFIG,
+  deserializeTransaction,
+} from "@railgun-community/shared-models";
+import { erc20ABI } from "@wagmi/core";
+import { ethers, constants } from "ethers";
+import { useState } from "react";
+import {
+  useAccount,
+  useSigner,
+  usePrepareContractWrite,
+  useContractWrite,
+  useProvider,
+} from "wagmi";
 
 // TODO this is an example list, update token list with real list of supported tokens
 const tokenList = [
   {
-    chainId: 1,
-    address: "0x111111111117dC0aa78b770fA6A738034120C302",
-    name: "1inch",
-    symbol: "1INCH",
+    chainId: 5,
+    address: "0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6",
+    name: "Wrapped ETH",
+    symbol: "WETH",
     decimals: 18,
-    logoURI: "https://assets.coingecko…nch-token.png?1608803028",
-  },
-  {
-    chainId: 1,
-    address: "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9",
-    name: "Aave",
-    symbol: "AAVE",
-    decimals: 18,
-    logoURI: "https://assets.coingecko…humb/AAVE.png?1601374110",
+    logoURI: "",
   },
 ];
 
 export const TxForm = () => {
-  // TODO: Placeholder notification for shielding
-  //
-  const { notifyUser } = useNotifications();
+  const network = NetworkName.EthereumGoerli;
```

Comment:
Maybe create a ticket to add this to the network global state.

Reply:
tracking here https://github.com/ScopeLift/railgun-shielder/issues/41